extensions/podman/packages/extension/src/utils/podman-cli.spec.ts

```diff
@@ -0,0 +1,184 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { type Configuration, env, process } from '@podman-desktop/api';
+import { beforeEach, describe, expect, test, vi } from 'vitest';
+
+import { isMultiplePodmanInstalled } from './podman-cli';
+
+const config: Configuration = {
+  get: vi.fn().mockReturnValue(undefined), // Default: no custom binary path
+  has: vi.fn().mockReturnValue(false),
+  update: vi.fn(),
+};
+
+// Mock external dependencies
+vi.mock('@podman-desktop/api', () => {
+  return {
+    configuration: {
+      getConfiguration: (): Configuration => config,
+    },
+    process: {
+      exec: vi.fn(),
+    },
+    env: {
+      isWindows: false, // Default to Linux/macOS
```

Comment:
please choose one isMac or isLinux to true, even if the code checks that it is not windows, it is not clear what we are testing here